mpd/base.py

```diff
@@ -404,8 +406,9 @@ def _parse_sticker(self, lines):
     def _parse_stickers(self, lines):
         return dict(self._parse_raw_stickers(lines))
 
-    def albumart(self, uri):
-        return self._execute_binary("albumart", [uri])
+    @mpd_commands("albumart", is_binary=True)
```

Comment:
Looks good. This should also work for `readpicture` the same way no?

Reply:
Should, yes; I don't have a sufficiently recent mpd around to test it with, but in theory adding `"readpicture"` should do the job. It'd report the binary like albumart, and possibly the mime type.